docs/usage_guide.rst

```diff
@@ -1866,6 +1866,11 @@ This can happen for many reasons. The asset may have been acquired in a non-supp
 
 The way to fix it is to add either a :ref:`manual trade<adding-manual-trade>` to tell rotki how you acquired that asset or an acquisition history event.
 
+#Error when importing CSV to Google Docs
+-------------------------------------------------
```

Comment:
```suggestion
Error when importing CSV formulas to Google Docs
-------------------------------------------------

```

Hey friend thanks! Since this is not markdown you would need to do the header as per my suggestion to render it properly